annotate.nf

```diff
@@ -177,28 +181,56 @@ process RunSnpeff {
   """
 }
 
-if (params.verbose) snpeffReport = snpeffReport.view {
+if (params.verbose) snpeffOutput = snpeffOutput.view {
   "snpEff report:\n\
   File  : ${it.fileName}"
 }
 
 // When we are running in the 'merge' mode (first snpEff, then VEP)
 // we have to exchange the channels
 
+process CompressSnpeffVCF {
+  tag {vcf}
+
+  publishDir directoryMap.snpeff, mode: 'link'
+
+  input:
+    set variantCaller, file(vcf) from snpeffVCF
+
+  output:
+    set variantCaller, file("*.vcf.gz") into snpeffVCFcompressed
+    file("*.vcf.gz.tbi")
+
+  script:
+  """
+  cat ${vcf} | bgzip > ${vcf}.gz
```

Comment:
Use `bgzip < ${vcf} > ${vcf}.gz` instead to avoid winning the “[Useless Use of Cat Award](http://porkmail.org/era/unix/award.html)” :smiley_cat:

Reply:
:man_facepalming: 
Thanks !!!